### PR TITLE
Fix membership import url

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
Overview
----------------------------------------
Fix membership import url

Affects 5.74rc, master

Before
----------------------------------------
The url in the navigation menu doesn't work

After
----------------------------------------
URL updated to one that does work

Technical Details
----------------------------------------
The form controller for these now expects standardising urls but only the participant one was fixed.

Note that they work even if not declared via xml but specific declaration does allow permissions to be defined

Comments
----------------------------------------

### CODE REVIEW

File: CRM/Upgrade/Incremental/php/FiveSeventyFour.php [hunk 1]
```diff
@@ -25,10 +25,15 @@ class CRM_Upgrade_Incremental_php_FiveSeventyFour extends CRM_Upgrade_Incrementa
    * Upgrade step; adds tasks including 'runSql'.
    *
    * @param string $rev
-   *   The version number matching this function name
+   *   The version number matUPDATE civicrm_navigation SET url = 'civicrm/import/participant?reset=1'
+WHERE url = 'civicrm/event/import?reset=1';ching this function name
```
Comment (composerjk) [inline CRM/Upgrade/Incremental/php/FiveSeventyFour.php:29]: @eileenmcnaughton Somehow some SQL got mixed into this comment?

**Comment by petednz** [comment]
Also note that the new page has a title that just says IMPORT - would be better imo if it said Membership Import

**Comment by eileenmcnaughton** [comment]
@petednz that's because you are accessing the right url without this code update I think (the new url already works but is not propertly registered without this)

File: CRM/Upgrade/Incremental/php/FiveSeventyFour.php (hunk 1 above)
Comment (eileenmcnaughton) [reply CRM/Upgrade/Incremental/php/FiveSeventyFour.php]: oops - fixed

**Comment by seamuslee001** [comment]
I've just pushed an update to fix up the civicrm_generated file as well

**Comment by eileenmcnaughton** [comment]
thanks @seamuslee001 - tests have passed now - OK to merge?

**Pull request merged**